apps/dailyscrum/src/app/sign-up/complete/complete-sign-up-form.tsx

```diff
@@ -0,0 +1,72 @@
+"use client";
+
+import { useForm } from "react-hook-form";
+import { Button } from "ui/button";
+import {
+  Form,
+  FormControl,
+  FormDescription,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from "ui/shadcn-ui/form";
+import { Input } from "ui/shadcn-ui/input";
+
+import { zodResolver } from "@hookform/resolvers/zod";
+
+import { z } from "zod";
+import { useTransition } from "react";
+import { completeSignUp } from "./actions";
+
+const formSchema = z.object({
+  name: z.string().min(1),
+});
+
+const CompleteSignUpForm = () => {
```

Comment:
A form component that appears after the signup. It's going to be kept visible to the user until a profile is created for the user.

<img width="546" alt="image" src="https://github.com/kiwiy-is/dailyscrum/assets/17868599/d1249b8c-b966-499f-823c-182f76f8b6f2">